GOVERNANCE.md

```diff
@@ -28,8 +26,14 @@ No more than one-fourth of the CC members may be affiliated with the same employ
 
 The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
```

Comment:
Ah, I see line 21 was moved down to line 35. We may want to note that line 35 is the definition of an "active member", if that's indeed the case.